### PR TITLE
Add MultipleChoiceField

### PR DESCRIPTION
**Pull request by softinio**
This is related to https://github.com/TrackMaven/trackmaven-drf/pull/31

Its the Django Custom field associated with it.

### CODE REVIEW

**Comment by softinio** [comment]
@KetsuN Can you QA this too as its related to my `trackmaven-drf` change and you have already looked at it?

**Comment by KetsuN** [comment]
@softinio Just to verify, this is a direct port from https://github.com/TrackMaven/TrackMaven/blob/master/trackmaven/apps/utils/fields.py#L25, right?

**Comment by softinio** [comment]
@KetsuN Yes its a direct port with no changes to the code. Only thing new I added is the unit tests as it had none originally.

File: tests/test_fields.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        self.assertEqual(f.get_prep_value('input1'), 'input1')
+        self.assertEqual(f.get_prep_value(['input1', 'input2', 'input4']),
+                         'input1,input2,input4')
+
+    def test_get_prep_lookup(self):
+        f = fields.MultipleChoiceField(
+                choices=VALID_CHANNEL_NAMES,
+                default='input1,input2,input4')
+        self.assertEqual(f.get_prep_lookup('exact', 'input1'), 'input1')
+        self.assertEqual(f.get_prep_lookup('contains', 'input1'), 'input1')
```
Comment (KetsuN) [inline tests/test_fields.py]: @softinio 

I'm not sure this assert statement accurately tests the `contains` logic.  I tried it locally changing this line to `self.assertEqual(f.get_prep_lookup('contains', 'put1'), 'input1')` and it fails.  I would expect that `input1` to contain `put1`.

Comment (softinio) [inline tests/test_fields.py]: @KetsuN 

I didn't write the original code, but if you look here it is behaving as expected returning the value passed in: 

https://github.com/TrackMaven/trackmaven-django-common/blob/dashboard-multiple-choice-field/trackmaven_django/fields.py#L67

Unless I have misunderstood the implication of this. 

We can get together to discuss?

**Comment by softinio** [comment]
Ready for final QA @KetsuN Thanks

**Comment by KetsuN** [comment]
@softinio I think this looks good!

**Pull request merged**